package.json

```diff
@@ -62,8 +66,8 @@
     "typescript": "3.5.2"
   },
   "peerDependencies": {
-    "@aws-cdk/aws-ecs": "^0.32.0 || ^0.33.0 || ^0.34.0 || ^0.35.0",
-    "@aws-cdk/aws-route53-targets": "^0.32.0 || ^0.33.0 || ^0.34.0 || ^0.35.0",
```

Comment:
I accidentally added this to peerDependencies at 3bc3f47f9999187b54994e8c5a2f2030fcd35e25 , but apparently this package does not need aws-route53-targets as peer.